desktop/hooks/UseContextMenu.ts

```diff
@@ -5,27 +5,28 @@
  * Copyright © 2022 Extremely Heavy Industries Inc.
  */
 import {XH} from '@xh/hoist/core';
-import {contextMenu as contextMenuEl} from '@xh/hoist/desktop/cmp/contextmenu/ContextMenu';
+import {contextMenu as contextMenuEl, ContextMenuItemLike} from '@xh/hoist/desktop/cmp/contextmenu/ContextMenu';
 import {ContextMenu} from '@xh/hoist/kit/blueprint';
 import {isArray, isFunction, isUndefined, isEmpty} from 'lodash';
 import {ReactElement} from 'react';
 import {cloneElement, isValidElement} from 'react';
 
+export type UseContextMenuSpec = ContextMenuItemLike[]|((e: MouseEvent) => ContextMenuItemLike[])|ReactElement;
```

Comment:
I find the name `UseContextMenuSpec` confusing.  Its only one argument to that hook, not a specification of the hook.   Can we call it `ContextMenuSpec`.